source/routes/read/user.js

```diff
@@ -0,0 +1,30 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+const authenticate = require(_base + 'middleware/authenticate');
+
+// Send in a valid json containing just the id and get back the document's json
+// Request json must be in the form:
+// {
+// "username": "the username of the user to retrieve",
+// }
+// TODO: Remove? All we're returning is the username which is also the index queried to get the info
+module.exports = {
+  '/read/user': {
+    methods: ['get'],
+    middleware: [authenticate],
+    fn: function(req, res, next) {
+      db.get(req.body.username)
+          .then((response) => {
+            response.username = response._id;  // Response body should use 'username' not '_id'

```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '// Response bo...'.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7405452)
```suggestion
            response.username = response._id; // Response body should use 'username' not '_id'
```